app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1 class="text-center mt-2">New recipe</h1>
+
+<%= render "form", recipe: @recipe %>
+
+<br>
+
+<div>
+  <%= link_to "Back to recipes", recipes_path %>
+</div>
```

Comment:
- [ ] I am not able to add a recipe and we are not able to view the list of recipes, view recipe details and add ingredient please fix that